docs/manuals/core/versioning/production_mode.md

```diff
@@ -18,76 +18,217 @@ To convert an experiment version to a production, you can run the Taipy applicat
+However, since there are conflicting changes between version 1.0 and 2.0, accessing an entity from version 1.0
+may lead to inconsistent behavior.
+
+It is recommended that when there are conflicting changes between production versions, migration functions should
+also be provided. These functions accept an entity as the input, and should return the newly migrated entity.
+
+From the log message, you can see that Taipy does not find any migration function from version 1.0 to 2.0.
+Let's create migration functions to make entities from production version 1.0 compatible with 2.0.
+
+```python
+def migrate_datanode_scope(datanode):
+    datanode.scope = Scope.GLOBAL
+    return datanode
+
+def migrate_skippable_task(task):
+    task.skippable = True
+    return task
+```
+
+In this example, we have 2 migration functions, `migrate_datanode_scope()` and `migrate_skippable_task()`.
+Notice that between version 1.0 and 2.0, each data node has its path and scope changed. However, we only migrate
+the scope because we want to keep the path pointing to the correct pickle file. You can freely modify the
+migration function as you wish, but be careful with how it may affect the application.
+
+To register the migration functions to Taipy, use the `Config.add_migration_function()^` method.
+This method requires the following parameters.
+
+- ***target_version*** represents the production version that entities are migrated to.
+
+- ***config*** indicates the configuration or the id of the config that needs to migrate.
+
+- ***migration_fct*** represents the migration function that takes an entity as input and returns a new entity
+  that is compatible with the target production version.
+
+```python
+Config.add_migration_function(
+    target_version="2.0",
+    config="input",
+    migration_fct=migrate_datanode_scope,
+)
+
+Config.add_migration_function(
+    target_version="2.0",
+    config="output",
+    migration_fct=migrate_datanode_scope,
+)
+
+Config.add_migration_function(
+    target_version="2.0",
+    config="example_algorithm",
+    migration_fct=migrate_skippable_task,
+)
+```
+
+After registering the migration functions, we can run the application again. Let's use the [scenario selector](../../gui/corelements/scenario_selector.md) to create a scenario for July.
+
+![July scenario in production version 2.0](pic/scenario_selector_for_production_ver2.png)
+
+Next, let's access June scenario.
+
+![Migrated June scenario in production version 1.0](pic/scenario_selector_for_production_ver1_migrated.png)
+
+We can see that the scope of the data nodes are updated to `GLOBAL`, and the skippable attribute of the task is
+also updated to True. Notice that since we don't migrate the path of the data nodes, the path of the data nodes
+are still pointing to the pickle files of June scenario. And finally, all entities are now version 2.0 and compatible
```

Comment:
"of June scenario" -> "of the scenario for the month of June" (or "of the June scenario" which is less clear).